cli/cfg/dump.go

```diff
@@ -44,7 +44,9 @@ func dumpConfiguration(writer io.Writer, cmdCtx *cmdcontext.CmdCtx,
 			writer.Write([]byte(cmdCtx.Cli.ConfigPath + ":\n"))
 		}
 	}
-	err := yaml.NewEncoder(writer).Encode(cliOpts)
+	encoder := yaml.NewEncoder(writer)
+	encoder.SetIndent(2)
+	err := encoder.Encode(cliOpts)
 	return err
```

Comment:
```suggestion
	return encoder.Encode(cliOpts)
```

Reply:
Now I bump only for the `cluster` as you suggested, so this change is not relevant.